modules/gitops-installing-argo-rollouts-cli-using-linux.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * argo_rollouts/using-argo-rollouts-for-progressive-deployment-delivery.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-installing-argo-rollouts-using-linux_{context}"]
+= Installing Argo Rollouts CLI using Linux
+
+For Linux, you can install Argo Rollouts CLI by performing the following steps.
```

Comment:
To improve the sentence a little bit: (I am not sure about using "following steps" if we first have prereqs:
```suggestion
If you are a Linux user, you can install Argo Rollouts CLI by performing the steps described in this procedure.
```

Reply:
Fixed.